Source/compositorclient/cmake/FindWesterosClient.cmake

```diff
@@ -0,0 +1,70 @@
+# - Try to find BroadCom nxserver library.
+# Once done this will define
+#  WESTEROS_FOUND - System has westeros
```

Comment:
This does not exists. Shouldn't it be WESTEROS_CLIENT_FOUND instead?

Reply:
updated